### DISCUSSION TITLE
Scrub controls

### DISCUSSION THREAD

**Discussion by matt-bishop-IMTBL** [Nanobot567/musik#2] [Ideas]
Hello - I love this tool so far, but struggle sometimes with there being no FF/rewind, or scrubbing. Not sure if it’s even possible but it would make it much nicer to use for me personally.

**Comment by Nanobot567** [comment]
Hey there! Thanks for giving musik a try :)

Yeah, fast forward and rewind used to exist, but it was way too buggy to be useful, so I removed it XD. It is definitely possible, I just need to sit down and work at it for a while. :sweat_smile: 

I do have a bunch of other projects in the oven at the moment, though, so this is going in my to do list but I might not be able to get to it anytime soon. Sorry to keep you waiting! /gen

Pull requests are open, however, so if you'd like to try your hand at implementing it yourself, go for it! (But good luck trying to understand this mess if you do, haha)
